src/CoreServiceProvider.php

```diff
@@ -69,7 +70,8 @@ public function boot()
         $this->commands([
             ComponentSettings::class,
             LanguagesSeed::class,
-            ScenariosLanguageUpdate::class
+            ScenariosLanguageUpdate::class,
+            StripScenariosTranslatableStrings::class
```

Comment:
Minor thing, add trailing comma to make next update cleaner in git diff

```suggestion
            StripScenariosTranslatableStrings::class,
```